aspnetcore/web-api/index.md

```diff
@@ -201,38 +201,6 @@ The `ValidationProblemDetails` type:
 * Provides a machine-readable format for specifying errors in web API responses.
 * Complies with the [RFC 7807 specification](https://tools.ietf.org/html/rfc7807).
 
-To change the default response type to `SerializableError`, apply the highlighted changes in `Startup.ConfigureServices`:
-
-::: moniker-end
```

Comment:
This `:::moniker-end` line will need to be restored. It's causing a build warning right now.

Reply:
@scottaddie  that was in the cors doc and I fixed that.